x-pack/legacy/plugins/maps/public/layers/styles/vector/vector_style.js

```diff
@@ -333,11 +333,10 @@ export class VectorStyle extends AbstractStyle {
 
     const data = styleMetaDataRequest.getData();
     const fieldMeta = dynamicProp.pluckStyleMetaFromFieldMetaData(data);
-
     return fieldMeta ? fieldMeta : fieldMetaFromLocalFeatures;
   };
 
-  _getFieldFormatter(fieldName) {
+  _getFieldFormatter = fieldName => {
```

Comment:
hah, you are changing this back to a lambda function after requesting to to just a regular function. That is why I ways just make them a lambda to begin with :)